Lib/test/test_concurrent_futures/executor.py

```diff
@@ -205,3 +219,16 @@ def test_free_reference(self):
             for _ in support.sleeping_retry(support.SHORT_TIMEOUT):
                 if wr() is None:
                     break
+
+    def test_swallows_falsey_exceptions(self):
+        # see gh-132063: Prevent exceptions that evaluate as falsey
+        # from being ignored.
+        # Falsey exceptions return 0 when `__len__` method is called,
+        # False when `__bool__` method is called.
+
+        msg = 'lenlen'
+        with self.assertRaisesRegex(FalseyLenException, msg):
+            self.executor.submit(raiser, FalseyLenException, msg).result()
+        msg = 'boolbool'
+        with self.assertRaisesRegex(FalseyBoolException, msg):
+            self.executor.submit(raiser, FalseyBoolException, msg).result()
```

Comment:
```suggestion
        msg = 'boolbool'
        with self.assertRaisesRegex(FalseyBoolException, msg):
            self.executor.submit(raiser, FalseyBoolException, msg).result()

        msg = 'lenlen'
        with self.assertRaisesRegex(FalseyLenException, msg):
            self.executor.submit(raiser, FalseyLenException, msg).result()
```

Just so that the classes are used in the same order they are declared (this really eases reading IMO)

Reply:
Okay